app/engine/engine.ts

```diff
@@ -17,6 +17,7 @@ enableMapSet();
 
 export enum ActionType {
   Join = "join",
+  Kick = "kick",
```

Comment:
I considered calling this "Leave", but since a kick can be initiated by *another* player, I went with kick. I'm open to "Leave" if folks think that is better.

Reply:
I think "kick" is better, since anyone can kick anyone else. You can also kick yourself